core_backend/app/question_answer/schemas.py

```diff
@@ -9,18 +9,28 @@
 
 class QueryBase(BaseModel):
     """
-    Pydantic model for query APIs
+    Question answering query base class.
```

Comment:
This is also rendered on Swagger UI (under Schemas section at the bottom)
Made changes wherever I saw fit

<img width="434" alt="image" src="https://github.com/user-attachments/assets/b85abc19-a7e8-4473-900d-7e3f5b59e0e8">